sdk/src/driftClient.ts

```diff
@@ -154,8 +159,12 @@ import { PythSolanaReceiver } from '@pythnetwork/pyth-solana-receiver/lib/idl/py
 import { getFeedIdUint8Array, trimFeedId } from './util/pythPullOracleUtils';
 import { isVersionedTransaction } from './tx/utils';
 import pythSolanaReceiverIdl from './idl/pyth_solana_receiver.json';
-import { asV0Tx, PullFeed } from '@switchboard-xyz/on-demand';
-import switchboardOnDemandIdl from './idl/switchboard_on_demand_30.json';
+import {
+	asV0Tx,
+	ON_DEMAND_DEVNET_PID,
```

Comment:
should you just put this on the config so you dont need to add the extra isDevnet flag?